docs/advanced/api/authentication.rst

```diff
@@ -0,0 +1,64 @@
+==============
+Authentication
+==============
+
+The Flocker REST API uses TLS to secure and authenticate requests.
+This ensures an API request is both encrypted, and verified to have come from an authorised user, while the corresponding response is verified to have come from the genuine cluster control service.
+
+Certificates are used for both client and server authentication, entirely replacing the use of usernames and passwords commonly used in HTTPS.
+
+Therefore to grant a user access to your cluster's REST API, you will need to use the ``flocker-ca`` tool, installed as part of the ``flocker-cli`` package, to generate a certificate and private key that is then given to the API end user.
+To give a user access to a cluster's REST API, use the ``flocker-ca`` tool to generate a certificate and private key for the user.
+The ``flocker-ca`` tool is installed as part of the flocker-cli package.
+If you have not already followed these steps, see the `flocker-node installation instructions <../../indepth/installation>`.
+
+Generate an API user certificate
+================================
+
+The CLI package includes the ``flocker-ca`` program which is used to generate certificate and key files.
+
+.. code-block:: console
+
+    $ flocker-ca --help
```

Comment:
Ditto about the `--help` changing rapidly. Maybe just remove it completely, or just mention "you can always run `flocker-ca --help`".